### DISCUSSION TITLE
nacos dubbo service ip变更后，higress服务列表中对应服务ip未同步

### DISCUSSION THREAD

**Discussion by adminhxx** [alibaba/higress#604] [Q&A]
会导致http转dubbo的请求失败
higress 1.2.0
nacos 2.2.0
dubbo 2.0.2

**Comment by CH3CHO** [comment]
能稳定重现吗？可以提供一下 controller Pod 中 higress-core 容器的日志吗？

**Comment by johnlanni** [comment]
@adminhxx 有更多信息么，比如上面说的higress-core的日志，以及nacos sdk的日志，可以进higress-core容器里到  /var/log/nacos/log 这个目录下看：
![image](https://github.com/alibaba/higress/assets/6763318/55e4e8f8-0ae6-42f4-8a20-eede4f93c859)

**Comment by johnlanni** [comment]
@adminhxx 1.3.1 版本做了推送优化，之前版本确实可能存在推送阻塞导致新的ip无法更新的问题
